src/gui/editors/PianoRoll.cpp

```diff
@@ -202,10 +202,10 @@ PianoRoll::PianoRoll() :
 	signalMapper = new QSignalMapper( this );
 	m_semiToneMarkerMenu = new QMenu( this );
 
-	QAction * markSemitoneAction = new QAction( tr("Mark/unmark current semitone"), this );
-	QAction* markScaleAction = new QAction( tr("Mark current scale"), this );
-	QAction* markChordAction = new QAction( tr("Mark current chord"), this );
-	QAction* unmarkAllAction = new QAction( tr("Unmark all"), this );
+	QAction *markSemitoneAction = new QAction( tr("Mark/unmark current semitone"), this );
+	QAction *markScaleAction = new QAction( tr("Mark current scale"), this );
+	QAction *markChordAction = new QAction( tr("Mark current chord"), this );
+	QAction *unmarkAllAction = new QAction( tr("Unmark all"), this );
```

Comment:
On 01/13/2015 06:32 PM, Jonas Trappenberg wrote:
>
> I would personally always put it on the right to avoid unclear
> declarations like
>
> char* a, b;

We have no official guideline on this - the asterisk position can be
decided by each programmer on their own.